frontend/main.js

```diff
@@ -430,7 +430,7 @@ async function startApp () {
         }
       },
       isInCypress () {
-        return !!window.Cypress
+        return !!window.Cypress && window.sbp === sbp
```

Comment:
But the problem is that `window.Cypress` is something that anyone can set through console. So I think we need to verify if it's real.
```
window.Cypress = true
```

Reply:
They can completely change the code on the frontend and there is nothing we can do to prevent that. The way we've designed the app to come to "consensus" among peers is how we deal with compromised clients. So the existing check we have is fine

Reply:
Okay, since it's for fixing Cypress errors (Issue #2058, and #2059), I will update it in the PR #2063.